types/package.json

```diff
@@ -6,6 +6,7 @@
   "repository": "https://github.com/desmos-labs/contracts-utils",
   "license": "Apache-2.0",
   "scripts": {
+    "pull-cw721-schemas": "ts-node src/pull-cw721-schemas.ts",
     "pull-schemas": "ts-node src/pull-schemas.ts",
```

Comment:
```suggestion
    "pull-schemas": "ts-node src/pull-schemas.ts && ts-node src/pull-cw721-schemas.ts",
```

Reply:
If we combine them to be one, then `--branch`, `--tag` and `commit` could not be used.
I would have both sperately commands and a merged command as follows:
- pull-contract-schemas
- pull-cw721-schemaa
- pull-schemas